packages/eslint-plugin/tests/docs.test.ts

```diff
@@ -113,7 +115,10 @@ describe('Validating rule docs', () => {
         // Rule title does not match the rule metadata.
         expect(tokens[1]).toMatchObject({
           type: 'paragraph',
-          text: `${rule.meta.docs?.description}.`,
+          text: `${rule.meta.docs?.description.replace(
+            /(?<!`)(require|enforce|disallow)/gi,
+            '$1s',
+          )}.`,
```

Comment:
Continuing the thread from https://github.com/typescript-eslint/typescript-eslint/pull/4976#discussion_r873180359: I'd really rather we didn't do the extra work to massage between one and two. In addition to requiring (admittedly clever) tricks like this one, I get a bit of cognitively dissonance having _slightly_ different descriptions for the same rule in multiple places.

It's fine having this here for now, so we can just move this discussion to #5096 as a followup if you feel strongly.

Reply:
I agree. I also mentioned in https://github.com/typescript-eslint/typescript-eslint/pull/5085#issuecomment-1139284871 I'd like this to be eventually automated as well